CMakeLists.txt

```diff
@@ -204,6 +204,8 @@ file(COPY third_party/OpenSSL/LICENSE DESTINATION ${CMAKE_BINARY_DIR}/licenses/o
 file(COPY third_party/qtpropertybrowser/LICENSE.txt DESTINATION ${CMAKE_BINARY_DIR}/licenses/qtpropertybrowser/licenses/)
 file(COPY third_party/conan/LICENSE.md DESTINATION ${CMAKE_BINARY_DIR}/licenses/conan/licenses/)
 file(COPY third_party/stb/LICENSE DESTINATION ${CMAKE_BINARY_DIR}/licenses/stb/licenses/)
+file(COPY third_party/vulkan/LICENSE.txt DESTINATION ${CMAKE_BINARY_DIR}/licenses/vulkan/licenses/)
+file(COPY third_party/vulkan-docs/LICENSE.adoc DESTINATION ${CMAKE_BINARY_DIR}/licenses/vulkan/licenses/)
```

Comment:
Why do we need the vulkan-docs license? Do we distribute anything from the docs?

Reply:
Thanks. I wasn't aware that it is only about the distribution.